app/src/components/market/common/set_allowance/index.tsx

```diff
@@ -45,11 +45,13 @@ export const SetAllowance: React.FC<SetAllowanceProps> = (props: SetAllowancePro
 
   return (
     <Wrapper {...restProps}>
-      <Title>Set Allowance</Title>
+      <Title>{collateral ? 'Set Allowance' : 'Enable Markets'}</Title>
       <DescriptionWrapper>
         <Description>
-          This permission allows the smart contracts to interact with your {collateral && collateral.symbol}. This has
-          to be done for each new token.
+          {collateral
+            ? `This permission allows the smart contracts to interact with your ${collateral &&
+                collateral.symbol}. This has to be done for each new token.`
+            : `This permission allows you to interact with market contracts.`}
```

Comment:
Yes that PR went from 1 file changed to 26 in a single commit because I kept the habit of checking out branches by passing from master and pulling all the changes. When working here on the refactoring issues I must not do it